### DISCUSSION TITLE
잘 밤에 쓸데없는 생각하기/맥에서 gpg 키 서명 실패

### DISCUSSION THREAD

**Discussion by giscus** [ujuc/ujuc.github.io#72] [General]
# 잘 밤에 쓸데없는 생각하기/맥에서 gpg 키 서명 실패

commit 할때마다 gpg키로 서명하게 해놨는데 안된다. 고치자!

https://ujuc.github.io/2020/02/19/macos-e-seo-gpg-ki-seo-myeong-sil-pae/

**Comment by ujuc** [comment]
https://youtrack.jetbrains.com/issue/IDEA-127802/Git-commit-fails-if-GPG-signing-is-enabled#focus=Comments-27-5069874.0-0

연관된 문서들을 묶어서 정리한번하자. 내가 쓴글도 잊어먹고 있었네...
